src/main/perf/Indexer.java

```diff
@@ -506,18 +506,12 @@ public DocValuesFormat getDocValuesFormatForField(String field) {
       throw new RuntimeException("exceptions during indexing");
     }
 
-    // Very tricky: if the line file docs source is binary, and you use multiple threads, and you use grouping fields, then the doc count
```

Comment:
Also evil past workaround comment!!  With the ominous `Very tricky` warning.

Reply:
Yeah glad I'm able to remove the "Very tricky" warning!